aip/0152.md

```diff
@@ -0,0 +1,84 @@
+---
+aip:
+  id: 152
+  state: reviewing
+  created: 2020-04-27
+permalink: /152
+redirect_from:
+  - /0152
+---
+
+# Jobs
+
+Occasionally, APIs may need to expose a task that takes significant time to
+complete, and where a transient [long-running operation][aip-151] is not
+appropriate. For example, a task could need to run repeatedly, or have separate
+permissions for configuring the task as opposed to running it.
+
+## Guidance
+
+An API **may** define a `Job` resource to represent a particular task with
+distinct setup, configuration, and execution:
+
+```proto
+message WritingJob {
```

Comment:
I'm worried that the -ing ending might hint that this is a naming convention for jobs -- which I don't believe it is. Do we have another example here?

Reply:
Maybe DuplicateBookJob?

Reply:
How about `WriteBookJob`.